global.R

```diff
@@ -1,3 +1,16 @@
+library(shiny)
+library(xlsx)
+library(ggplot2)
+library(DT)
+library(beeswarm)
+library(stringr)
+library(pheatmap)
+library(readxl)
+library(gplots)
+library(RColorBrewer)
+library(plyr)
+library(ComplexHeatmap)
+
```

Comment:
happy to see all these consolidated, just want to make sure all the functionality of the app is still the same as a result (it's sad but package loading order matters in R). Looking forward to seeing a PRoXe_alpha version!

Reply:
I'm uploading PRoXe_alpha now, so once that's up I'll make sure everything is working.

Rather than depending on packages being loaded in a specific order, I'd suggest handling conflicts by using the `package::function()` method or the [conflicted package](https://github.com/r-lib/conflicted) (which essentially just says which package's version of a function to use throughout a script).

Reply:
i'm running into some bioconductor repo/package issues for uploading the app, i'll take a closer look tomorrow! not that you'd be wanting to review this on a Friday night anyways :)

Reply:
Yep! `package::function()` is certainly best practice.